corehq/util/couchdb_management.py

```diff
@@ -59,5 +65,11 @@ def get_db_for_class(self, klass):
     def get_db_for_doc_type(self, doc_type):
         return Database(self.get_db_uri_for_doc_type(doc_type))
 
+    def get_db_for_postfix(self, postfix):
+        return Database(self.all_db_uris_by_slug[postfix])
```

Comment:
yeah i did add it to remove `create=True` (which hangs on tests if the name is bad) but i was only using it for tests so I just got rid of it for now